tools/f18/CMakeLists.txt

```diff
@@ -26,8 +21,6 @@ target_link_libraries(f18
   FortranParser
   FortranEvaluate
   FortranSemantics
-  ${FORTRAN_FIR_LIB}
-  ${LLVM_COMMON_LIBS}
```

Comment:
If `LLVM_COMMON_LIBS` is going to be needed for the FIR replacement I think it would be fine to leave it in, even though it's not currently necessary to be able to link.

Reply:
I don't think it serves any purpose to leave it in. LLVM is unused by the f18 program and not needed for linking it. 

The new middle end will explicitly require sets of MLIR and LLVM libraries in its cmake specs.